### DISCUSSION TITLE
一个疑为因telegram对通知模板中某些字符无法解析引发的错误

### DISCUSSION THREAD

**Discussion by voidoath** [walinejs/waline#904] [Q&A] [answered]
如图，使用默认通知模板，在评论时弹出以下错误提示

![Snipaste_2022-04-09_13-55-25](https://user-images.githubusercontent.com/53803262/162558748-4763821f-050f-4634-8622-87107e20660a.png)

将 `TG_TEMPLATE` 改为‘test’后，可以正常评论，并能接收到tg bot发送的‘test’消息

请问如何修改通知模板，使其不引发错误，且能显示新评论的文章及链接？

**Comment by Mister-Hope** [comment, marked as answer]
cc @lizheming  这个有check么？

**Reply by lizheming** [reply]
这个不好check，建议引导用户自定义模板

**Reply by Mister-Hope** [reply]
我看他说的意思是我们的默认模板有问题？

**Reply by lizheming** [reply]
默认模板没有问题，是因为动态内容导致的问题

**Comment by Suleman-Elahi** [comment]
Getting the same error.

`Telegram Notification Failed:{"ok":false,"error_code":400,"description":"Bad Request: can't parse entities: Character '.' is reserved and must be escaped with the preceding '\\'"}`

Using the same template on the website.

💬 _[{{site.name}}]({{site.url}}) has a new comment

_{{self.nick}}_ replied:

```
{{self.comment-}}
```
{{-self.commentLink}}
Email: _\`{{self.mail}}\`
Audit: _{{self.status}}

It is only for comment preview, click [view full content]({{site.postUrl}})

Approve this comment here: [bitly](https://www.bitlylinks.com/8zikcLWEJ)

**Reply by lizheming** [reply]
Telegram has certain requirements on the grammar of the content. If some special strings are included in the comment content, telegram's API will return an error. If you are very concerned about this problem, it is recommended that you use the `TG_TEMPLATE` environment variable to set the notification template a little simpler, such as the stable content of "A new comment!", or replace it with other notification tool.

**Reply by Suleman-Elahi** [reply]
I will try.
